pkg/rpc/wallet.go

```diff
@@ -258,3 +258,34 @@ func (s *WalletService) SendTransaction(opts *SendTransactionOptions) (*SendTran
 
 	return r, resp, nil
 }
+
+// SendCatTransactionOptions represents the options for send_transaction
+type SendCatTransactionOptions struct {
+	WalletID uint32 `json:"wallet_id"`
+	Amount   uint64 `json:"amount"`
+	Address  string `json:"inner_address"`
+	Fee      uint64 `json:"fee"`
+}
+
+// SendCatTransactionResponse represents the response from send_transaction
+type SendCatTransactionResponse struct {
+	Success       bool                    `json:"success"`
+	TransactionID string                  `json:"transaction_id"`
+	Transaction   types.TransactionRecord `json:"transaction"`
+}
+
+// SendCatTransaction sends a transaction
+func (s *WalletService) SendCatTransaction(opts *SendCatTransactionOptions) (*SendCatTransactionResponse, *http.Response, error) {
```

Comment:
It looks like other functions in this package use the names of the actual RPC endpoint. In this case it would be `cat_spend`. Based on that, it would seem that `CatSpend` should be the function name (based on the current patterns.)

This would also change the names of the structs associated with this function, if we agree.